digital-signature/README

```diff
@@ -0,0 +1,6 @@
+# sender
```

Comment:
I think this code should probably be moved into the security module - https://github.com/eugenp/tutorials/tree/master/libraries-security